main/dzVents.cpp

```diff
@@ -117,25 +117,14 @@ void CdzVents::ProcessHttpResponse(lua_State *lua_state, const std::vector<CEven
 						lua_pushstring(lua_state, (*itt2).substr(pos + 2).c_str());
 						lua_rawset(lua_state, -3);
 					}
-					else
-					{
-						size_t pos = (*itt2).find("HTTP/");
-						if (pos != std::string::npos)
-						{
-							lua_pushstring(lua_state, "status");
-							lua_pushstring(lua_state, (*itt2).c_str());
-							lua_rawset(lua_state, -3);
-						}	
-					}	
 				}
-				// last item in vector is status code when method = GET
-				itt2 = itt->vData.end() - 1;
-				std::stringstream ss(*itt2);
-				ss >> statusCode;
+				size_t pos = (*itt->vData.begin()).find("HTTP/");
+				if (pos != std::string::npos)
```

Comment:
I would compare it against 0 as it needs to be at the beginning

if (pos != 0)

Reply:
Thx !
Will do